frontend/src/css/modules/my-challenge-all-submission.scss

```diff
@@ -0,0 +1,9 @@
+/* Styles for View all submissions page */
+
+.all-submission-table {
+	width: 160%;
```

Comment:
@aayusharora Yes, the width can be more than `100%`. Since we are using horizontal scroll so to make the page less compact I have increased its width. Please refer [here](https://stackoverflow.com/questions/23934695/css-width-above-100).